doc/changes/latest.rst

```diff
@@ -53,3 +53,4 @@ Changes
 - Refactors fmriprep confound loading such that that the parsing of the 
   relevant image file and the loading of the confounds are done in 
   separate steps (:gh:`3274` by `David G Ellis`_).
+- Boolean input data in :func:`~image.new_img_like` now defaults to `np.uint8` instead of `np.int8`
```

Comment:
```suggestion
- Boolean input data in :func:`~image.new_img_like` now defaults to `np.uint8` instead of `np.int8` (:gh:`3286` by `Yasmin Mzayek`_).
```